test/iron-location.html

```diff
@@ -134,6 +149,38 @@
         assertHaveSameUrls(urlElem, otherUrlElem);
       });
     });
+    suite('when we do a synchronous redirection', function() {
+      test('of the hash', function() {
```

Comment:
The organization of tests is good here. However, if I read the output of the test runner when a test fails, it helps a lot to read a string that says what expectation was violated.

No action required, but please consider describing the expectation in a sentence here.